modules/libcom/src/osi/os/RTEMS/osdTime.cpp

```diff
@@ -83,7 +94,12 @@ void osdNTPInit(void)
 {
     struct sockaddr_in myAddr;
 
+#if __RTEMS_MAJOR__ > 4
+    ntpSocket = socket( AF_INET, SOCK_DGRAM, IPPROTO_UDP );
+#else
     ntpSocket = socket (AF_INET, SOCK_DGRAM, 0);
+#endif
```

Comment:
Check whether this `#if` is needed, `IPPROTO_UDP ` should work on all RTEMS versions, explain why it's needed in a comment if it doesn't.

Reply:
I have removed it, it is probably not necessary? In other implementations e.g. vxWorks the socket is not created at all, and also in  rtems_bsdnet_get_ntp() a socket is created if it does not exists already. Probably it is used here to be able to delete old answers of the NTP servers before the request.

I also took the opportunity to add a little information about the socketNr and the NTP server in osdNTPReport:

DYCRVME> NTPTime_Report
NTP driver is synchronized with server
Syncronization interval = 60.0 seconds
Last synchronized at 2023-01-05 13:38:40.114914
Current OS tick rate = 100 Hz
Measured tick rate = 0.000 Hz
osdNTPReport: ntpSocket used : 28
osdNTPReport: NTP Server = 10.0.0.1